src/lib/hooks/useUpdateGroups.ts

```diff
@@ -0,0 +1,52 @@
+import { useContext } from 'react';
+import { FirebaseContext } from '../../components/Firebase';
+import { GroupType } from '../types';
+
+export default function(options?: { client?: typeof firebase }) {
```

Comment:
Should we type the return value given that it's a `Promise` or is that inferred?

Reply:
Should be inferred